src/components/forms/profile/update-user-address-form.tsx

```diff
@@ -39,8 +43,23 @@ const UpdateUserAddressForm: React.FC<{
       <Form
         formID={formID}
         onSubmit={async () => {
-          setFormID(uuidv4());
-          return;
+          await fetchJson(
+            `/api/profile/addresses/${updatedAddress.id}`,
+            "PATCH",
+            {
+              userAddressPayload: updatedAddress,
+            },
```

Comment:
I think we can only send the payload directly without embedding it in an object:
```suggestion
            updatedAddress,
```
[1/2]